R/chrome.R

```diff
@@ -95,10 +95,13 @@ chrome_print = function(
   ))
 
   debug_port = servr::random_port()
+  log_file = if (isTRUE(getOption('pagedown.keep.chrome.log'))) {
```

Comment:
Just a style suggestion (feel free to ignore):

```suggestion
  log_file = if (getOption('pagedown.chrome.log', FALSE)) {
```